runtime/nls/shrc/j9shr_ca.nls

```diff
@@ -6919,7 +6919,7 @@ J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1.system_action=The JVM continues, but with memor
 J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1.user_response=No action required
 # END NON-TRANSLATABLE
 
-J9NLS_SHRC_CM_PRINTSTATS_PAGESIZEE=Mida de la p\u00e0gina de mem\u00f2ria = %zu (mida de la p\u00e0gina actual del sistema operatiu %zu)
+J9NLS_SHRC_CM_PRINTSTATS_PAGESIZEE=Mida de la p\u00e0gina de mem\u00f2ria        = %zu (mida de la p\u00e0gina actual del sistema operatiu %zu)
```

Comment:
I think one more space is needed for alignment, or is one of those unicode characters the width of two?

Reply:
I aligned it with the previous field J9NLS_SHRC_CM_PRINTSTATS_FEATURE in the same language.

```
Funció                              = %s
Mida de la pàgina de memòria        = %zu (mida de la pàgina actual del sistema operatiu %zu)
```